src/wp-admin/includes/class-wp-debug-data.php

```diff
@@ -839,6 +839,21 @@ public static function debug_data() {
 			);
 		}
 
+		// Server time
+		$date = new \DateTime( 'now', new \DateTimeZone( 'UTC' ) );
```

Comment:
Can we use `current_time` instead?

Reply:
We could use `current_time( DateTime::ATOM, true )`, and `current_time( DateTime::RFC850, true )` yeah. This would mean running the various conditions in `current_time()` for each call, and it's possible on a rare occasion that separate calls could produce different results (if the first occurs just before a tick).

Are you thinking to use `current_time()` for readability, or for other reasons?

Reply:
Indeed, I don't see much value except for readability… I'm not convinced it is the best approach for this specific use case